src/components/eventPage/eventPage.tsx

```diff
@@ -12,7 +12,7 @@ export const EventPage = async ({ slug }: Props) => {
   if (!data) return notFound()
 
   return (
-    <article className="content-layout">
+    <article>
       <h1 className="sr-only">{data.title}</h1>
```

Comment:
why sr-only?

Reply:
The header comes from Dato without the ability to add h1's. That's to make sure nobody is going to add two h1's on one page. But still a page needs an h1 so I hard coded that one to make sure a page has a h1